libraries/botbuilder/src/teamsInfo.ts

```diff
@@ -81,6 +82,31 @@ export class TeamsInfo {
         });
     }
 
+    /**
+     * Gets the information for the given meeting id.
+     * @param context The [TurnContext](xref:botbuilder-core.TurnContext) for this turn.
+     * @param meetingId The BASE64-encoded id of the Teams meeting.
+     * @returns The [TeamsMeetingInfo](xref:botbuilder-core.TeamsMeetingInfo) fetched
+     */
+    public static async getMeetingInfo(context: TurnContext, meetingId?: string): Promise<TeamsMeetingInfo> {
+        if (!context) {
+            throw new Error('context is required.');
+        }
+
+        const activity = context.activity;
+
+        if (meetingId == null) {
+            const meeting = teamsGetTeamMeetingInfo(activity);
+            meetingId = meeting ? meeting.id : undefined;
```

Comment:
You can write this as `meetingId = meeting?.id;`

Reply:
Ah yeah, I should've caught that. Fixed it here as well as the place I copy/pasted that piece from.